lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
         for filetype, ext in sorted(FigureCanvasWebAggCore.
                                     get_supported_filetypes_grouped().
                                     items()):
-            extensions.append(ext[0])
+            if not ext[0] == 'pgf':  # pgf does not support BytesIO
```

Comment:
Yes, I think this should be back ported

On Thu, Jan 8, 2015, 07:28 Michael Droettboom <notifications@github.com>
wrote:

> In lib/matplotlib/backends/backend_webagg_core.py
> <https://github.com/matplotlib/matplotlib/pull/3981#discussion-diff-22648899>
> :
>
> > @@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
> >          for filetype, ext in sorted(FigureCanvasWebAggCore.
> >                                      get_supported_filetypes_grouped().
> >                                      items()):
> > -            extensions.append(ext[0])
> > +            if not ext[0] == 'pgf':  # pgf does not support BytesIO
>
> If possible, I think it's preferable to fix this at the source (having pgf
> support BytesIO objects, if that's possible), rather than papering around
> it here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3981/files#r22648899>.
>